src/contracts/Chef.sol

```diff
@@ -116,4 +117,14 @@ abstract contract Chef is Ownable {
 
     event Harvest(address harvester, uint256 agIndex, uint256 amount);
     event Withdraw(address withdrawer, uint256 agIndex, uint256 amount);
+
+    //------------------------------------------------------Support Functions
+
+    struct AllocationGroup {
```

Comment:
why did ya move this here?  StakingChef doesn't need it, right?

Reply:
oh whoops that was an accident